installer/windows_exporter.wxs

```diff
@@ -23,7 +23,7 @@
     <SetProperty Id="ExtraFlags" After="InstallFiles" Sequence="execute" Value="[EXTRA_FLAGS]">EXTRA_FLAGS</SetProperty>
 
     <Property Id="LISTEN_ADDR" Secure="yes" />
-    <Property Id="LISTEN_PORT" Secure="yes" Value="9182" />
+    <Property Id="LISTEN_PORT" Secure="yes" />
     <SetProperty Id="ListenFlag" After="InstallFiles" Sequence="execute" Value="--telemetry.addr [LISTEN_ADDR]:[LISTEN_PORT]">LISTEN_ADDR OR LISTEN_PORT</SetProperty>
```

Comment:
It's been a while, but I think the reason it was defaulted is because otherwise if you *only* specify `LISTEN_ADDR`, it'll set the flag to `--telemetry.addr some-addr-specified:`, ie without the port number set. That's not a valid listening specifier, so we'd crash.
I think we'll need to do something clever/ugly with an additional property and changing the condition for `ListenFlag` in order to not set the flag unless something is specified. We have three cases:
- Neither port nor addr given -> Should not set flag
- Port given (addr given or not does not matter) -> Should set flag to addr:port (empty addr is fine)
- Addr given (but not port) -> Should set flag to addr:default_port

Reply:
I had a feeling it wouldn't be that easy. I'll have a look to see if I can address those cases.